### PR TITLE
Addresses handful of bugs

### PR DESCRIPTION
**Pull request by osahyoun**

### CODE REVIEW

File: members-service/triggerChampaignMemberUpdate.js [hunk 1]
```diff
@@ -10,18 +10,26 @@ const logger = new OperationsLogger({
...
 });
 
-export async function handler(e, ctx, cb, fn = updateMember) {
-  const [item] = e.Records;
+async function update(item, fn = updateMember) {
   const record = Converter.unmarshall(item.dynamodb.NewImage);
-  if (!updateMemberEvent(record)) return cb(null, 'Not a member update event');
+
+  if (!updateMemberEvent(item.eventName, record)) {
+    console.error('ERROR: Not a member update event');
```
Comment (rodrei) [inline members-service/triggerChampaignMemberUpdate.js]: Is this an actually an error?
If I understand correctly this function will be triggered whenever a new record is written in the 'operations table'. So, if a record that's not of type "updateMemberEvent" is added, we'd be logging an Error, when actually nothing has gone wrong.

**Comment by rodrei** [comment]
@osahyoun Next time can you please include some description of the stuff you're fixing please?

File: members-service/triggerChampaignMemberUpdate.js [hunk 2]
```diff
@@ -10,18 +10,26 @@ const logger = new OperationsLogger({
...
     logger.updateStatus(record, { champaign: 'SUCCESS' });
-    return cb(null, result);
   } catch (e) {
     logger.updateStatus(record, { champaign: 'FAILURE' });
-    cb(e);
-    throw e;
   }
 }
+
+export async function handler(e, ctx, cb, memberUpdater = updateMember) {
```
Comment (rodrei) [inline members-service/triggerChampaignMemberUpdate.js]: What about using more explicit variable names?

**Comment by rodrei** [comment]
Other than my comment, the code looks good! I say merge it!

**Comment by osahyoun** [comment]
Thanks, @rodrei.

**Pull request merged**